### ISSUE TITLE
Error: Unable to load schem from file "src/lambda/schema.ts" due to import error:

### ISSUE THREAD

**Issue by RobertBroersma** [dotansimha/graphql-code-generator#1948]
**Describe the bug**
I'm trying to import my schema from a TypeScript file, but I get an error when trying to generate. It seems like something is going wrong in the TS compilation.

Error:
```
  × ./src/generated/graphql-server.ts
    AggregateError:
        Error: Unable to load schem from file "src/lambda/schema.ts" due to import error: SyntaxError: Unex
pected identifier
            at SchemaFromExport.canHandle (C:/Home/repos/qadash/packages/dashboard/node_modules/graphql-too
lkit/src/loaders/schema/schema-from-export.ts:41:37)
        C:/Home/repos/qadash/packages/dashboard/src/lambda/schema.ts:5
        import gql from 'graphql-tag';
               ^^^
        SyntaxError: Unexpected identifier
            at Module.m._compile (C:/Home/repos/qadash/packages/dashboard/node_modules/ts-node/src/index.ts
:439:23)
            at Object.require.extensions.(anonymous function) [as .ts] (C:/Home/repos/qadash/packages/dashb
oard/node_modules/ts-node/src/index.ts:442:12)
```

**To Reproduce**

1. My GraphQL schema:

```graphql
import gql from 'graphql-tag'
import { typeDefs as PrismaSchema } from '../generated/prisma-client/prisma-schema'

const typeDefs = gql`
  type Query {
    repos: [Repo!]!
    repo(name: String): Repo!
    branches(repoName: String): [Branch!]!
    branch(name: String): Branch!
    deploys: [Deploy!]!
  }

  type Mutation {
    upsertDeploy(
      repoName: String!
      branchName: String!
      deployId: String!
      url: String
      status: DeployStatus
    ): Deploy!
  }
`

export default [PrismaSchema, typeDefs]
```

3. My `codegen.yml` config file:

```yml
overwrite: true
schema: src/lambda/schema.ts
maybeValue: 'T | undefined | null'
generates:
  ./src/generated/graphql-server.ts:
    config:
      contextType: ../lambda/context#MyContext
      mappers:
        Deploy: ./prisma-client#Deploy
        Branch: ./prisma-client#Branch
        Repo: ./prisma-client#Repo
    plugins:
      - 'typescript'
      - 'typescript-operations'
      - 'typescript-resolvers'
  ./src/generated/graphql.tsx:
    plugins:
      - 'typescript'
      - 'typescript-react-apollo'
  ./graphql.schema.json:
    plugins:
      - 'introspection'
require:
  - ts-node/register

```

**Environment:**

- OS: Windows 10
- `@graphql-codegen/cli@1.2.0`: 
- NodeJS: v10.15.1

**Comment by dotansimha** [comment]
Hi @RobertBroersma ! 
We are using `graphql-toolkit` for loading the schema, and if you are using a code file, you should `export default` a valid `GraphQLSchema` or `DocumentNode`.
In you code example, you are exporting a `[string, string]` so that's why it does not work for you.
You should either merge the ASTs into a single ast (with `concatAst`), or compile the schema with `buildSchema` (or `makeExecutableSchema`).

**Comment by RobertBroersma** [comment]
Hey @dotansimha, thanks for the reply. I understand that my export is invalid, I've tried and succeeded using `buildSchema`, however not using `export default` (I get an `unexpected token. export` instead of this one). I had to use `require()` instead of `import` and `module.exports` instead of `export default`, so I assume the error is within TypeScript compilation.

I've verified that
```
require:
  - ts-node/register
```
is loaded, so I'm not sure where to go from here! Running `ts-node` on my schema also seems to work fine.
